plugins/tiddlywiki/katex/wrapper.js

```diff
@@ -70,6 +77,23 @@ KaTeXWidget.prototype.render = function(parent,nextSibling) {
 	// Insert it into the DOM
 	parent.insertBefore(span,nextSibling);
 	this.domNodes.push(span);
+	var tiddlerFrame;
+	span.querySelectorAll('.katex-eqref [data-katex-label]:not([data-katex-eqnum])').forEach(element => {
+		// the tiddlyway requires a leading hash to jump inside a tiddler
+		var katexLabel = this.document.getElementById('#' + element.dataset.katexLabel);
```

Comment:
It looks like this solution uses DOM IDs to identify KaTeX formulae. The core avoids using IDs because the restriction a given ID can only be used once is problematic when in TiddlyWiki it is possible to open a tiddler more than once. For example, given a tiddler "Formulae" that contains a bunch of KaTeX formulae, it would be possible to both open that tiddler and alongside it display another tiddler that transcludes "Formulae".